tester/src/test/java/com/silanis/esl/sdk/examples/CreateAndUpdateSubAccountExampleTest.java

```diff
@@ -0,0 +1,43 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.api.model.AccessibleAccountResponse;
+import com.silanis.esl.api.model.Account;
+import com.silanis.esl.sdk.internal.EslServerException;
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.List;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+import static com.silanis.esl.sdk.examples.CreateAndUpdateSubAccountExample.NAME;
+
+public class CreateAndUpdateSubAccountExampleTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
Is this needed if we don't expect any exception ?

Reply:
You're right, actually, it's useless, will be removed.